website/docs/src/content/index.mdx

```diff
@@ -42,6 +42,76 @@ LLM workflows are fundamentally about function composition and data flow - exact
 
 While GenSX uses JSX, it does not share the same execution model as React, and has zero dependencies.
 
+## GenSX Cloud
+
+[GenSX Cloud](/docs/cloud) provides everything you need to ship production grade agents and workflow including a serverless runtime designed for long-running workloads, cloud storage to build stateful workflows and agents, tracing and observability, and features for team collaboration.
+
+### Serverless deployments
+
+Deploy any workflow to a hosted API endpoint with a single command. The GenSX cloud platform handles scaling, infrastructure management, and API generation automatically:
+
+```bash
+# Deploy your workflow to GenSX Cloud
+$ npx gensx deploy ./src/workflows.tsx
+✔ Building workflow using Docker
+✔ Generating schema
+
+✔ Successfully built project
+
+ℹ Using project name from gensx.yaml: support-tools
+✔ Deploying project to GenSX Cloud (Project: support-tools)
+
+✔ Successfully deployed project to GenSX Cloud
+
+Dashboard: https://app.gensx.com/gensx/support-tools/default/workflows
+
+Available workflows:
+- ChatAgent
+- TextToSQLWorkflow
+- RAGWorkflow
+- AnalyzeDiscordWorkflow
+
+Project: support-tools
+```
+
+The platform is optimized for AI workloads with millisecond-level cold starts and support for long-running executions up to an hour.
+
+### Development Server
+
+Test your workflows locally with a development server that matches the cloud environment:
+
+```bash
+# Start a local API server
+gensx start src/workflows.ts
+```
```

Comment:
imo we should replace this with a short section on tracing. I don't think the dev server needs real estate here (it's a utility feature rather than a game changer). Plus your thesis mentions tracing rather than the dev server :)